src/api/courses.js

```diff
@@ -34,30 +33,13 @@ const getCsv = questions => {
     'queue.Queue_CreatedAt',
     'queue.course.CourseName',
   ])
-  const timeFields = new Set([
-    'queue.Queue_CreatedAt',
-    'enqueueTime',
-    'dequeueTime',
-    'answerStartTime',
-    'answerFinishTime',
-  ])
 
   // Taken from https://stackoverflow.com/questions/8847766/how-to-convert-json-to-csv-format-and-store-in-a-variable
   const header = Array.from(columns)
   const replacer = (key, value) => (value === null ? '' : value)
   const csv = questions.map(row =>
     header
       .map(fieldName => {
-        if (timeFields.has(fieldName)) {
```

Comment:
cc @jackieo5023 - with UBC now using the queue, I don't think we want to localize times to Central timezone. I think it makes more sense to keep times in UTC and allow consumers to adjust to their timezones during processing if they want. Let me know if that makes sense to you.